  workaround for http://bugs.python.org/issue17143 (PY-8706)  